### PR TITLE
Save tour via API

### PR DESCRIPTION
**Pull request by jonw-cogapp**
- **Pivotal ticket**: [#186172083](https://www.pivotaltracker.com/story/show/186172083)

# What does this Pull Request do?

Gathers information from the App and supplies it in a post request to the API. The response from the server is then presented on the front-end

# Workflow

- [x] Before merging: After all review changes I've synchronised `develop` and rebased this branch on top of it

### CODE REVIEW

File: src/components/submission/Submission.jsx [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
     setValidityIssues,
     limits,
+    isSaving,
+    setIsSaving,
   } = useContext(AppContext);
+  const [saveResponse, setSaveResponse] = useState(null);
+
+  const handleSave = () => {
+    setIsSaving(true);
+    setTimeout(async () => {
```
Comment (hannahb-cogapp) [inline src/components/submission/Submission.jsx:23]: Remove setTimeout, only needed for testing

File: src/components/submission/Submission.jsx [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+          headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            title: tourTitle,
+            description: tourDescription,
+            artworks: tourItems.map((item) => ({
+              objectNote: item.note,
+              ...item,
```
Comment (hannahb-cogapp) [inline src/components/submission/Submission.jsx:37]: Look into whether this duplicates the note field

**Review by hannahb-cogapp — APPROVED** [review]
A couple of minor things to look into/tweak, but otherwise looks good.

File: src/components/submission/Submission.jsx (hunk 2 above)
Comment (jonw-cogapp) [reply src/components/submission/Submission.jsx]: You were correct, I've fixed this!

**Pull request merged**